src/loader.rs

```diff
@@ -256,6 +275,7 @@ impl AssetLoader<Map> for TiledMapLoader {
             meshes,
             layers,
             tile_size,
+            image_folder: asset_path.parent().unwrap().to_str().unwrap().to_string(),
```

Comment:
Should refactor all the error handling in a later PR to use the `?` operator (https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html#a-shortcut-for-propagating-errors-the--operator)